app/inputs/select_with_modal_help_input.rb

```diff
@@ -1,11 +1,4 @@
 class SelectWithModalHelpInput < MultiValueWithHelpInput
```

Comment:
Probably not. I am currently investigating what views in CC and Sufia need to be altered when we remove these special classes.